app.js

```diff
@@ -54,51 +54,44 @@ app.get('/examples*', function (req, res) {
 // Components
 app.get('/components*', function (req, res) {
   var path = (req.params[0]).replace(/\//g, '')
+  // If it isn't the isolated preview, render the component "detail" page
+  if (path.indexOf('preview') == '-1') {
```

Comment:
- 1 doesnt need to be a string, does it?

Reply:
Oops, will fix!